android/src/main/java/ee/forgr/audio/NativeAudio.java

```diff
@@ -222,6 +232,14 @@ public void configure(PluginCall call) {
         call.resolve();
     }
 
+    /**
+     * Checks whether an audio asset identifier has been preloaded.
+     *
+     * Resolves the provided PluginCall with a JSObject containing the boolean property
+     * `"found"` set to `true` when the given `assetId` is present in the plugin's preload map,
+     * or `false` when it is not present. Rejects the call with `ERROR_AUDIO_ID_MISSING` when
+     * the `assetId` parameter is missing or invalid.
+     */
```

Comment:
You are breaking the convention of Capgo plugins by adding these Javadocs. Furthermore, your Javadocs are simply incorrect as Javadocs are not markdown.

```suggestion
    /**
     * Checks whether an audio asset identifier has been preloaded.
     *
     * Resolves the provided PluginCall with a JSObject containing the boolean property
     * <code>found</code> set to <code>true</code> when the given <code>assetId</code> is present in the plugin's preload map,
     * or <code>false</code> when it is not present. Rejects the call with <code>ERROR_AUDIO_ID_MISSING</code> when
     * the <code>assetId</code> parameter is missing or invalid.
     */
```